pkg/api/types/field_type.go

```diff
@@ -83,19 +105,8 @@ type FieldTypeSingleSelect struct {
 // For example, given a form "Projects", this form could have a subform "Monthly Deliveries".
 // The "Monthly Deliveries". There could be multiple "Monthly Deliveries" for a single "Project".
 type FieldTypeSubForm struct {
-	// ID represents the ID of the sub form
-	ID string `json:"id" yaml:"id"`
-	// Name represents the Name of the sub form
-	Name string `json:"name" yaml:"name"`
-	// Code represents the unique Code for the subform Field
-	Code string `json:"code,omitempty" yaml:"code,omitempty"`
```

Comment:
We remove subform id, name and code. Only `fields` is now part of the `SubForm`. It will inherit the name and if from the `field`